hooks/pre/31-mongo_storage_engine.rb

```diff
@@ -0,0 +1,56 @@
+MONGO_DB_NAME = pulp_db_param.value
```

Comment:
This is not defined yet and errors. I'd move this to line 18/19 and lowercase the MONGO_DB_NAME variable.